### PR TITLE
Add keystore to cloudant FAT

### PR DESCRIPTION
**Pull request by tkburroughs**
Precreate the keystore for a cloudant FAT to avoid CWPKI0804E that occurs when automatic creation intermittently fails.
Updated to test to no longer wait for keystore create message; doesn't occur since pre-created
Updated buildpath for dependency in build.gradle that is needed to run locally

- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.ws.rest.handler.validator.cloudant_fat/bnd.bnd [hunk 1]
```diff
@@ -27,6 +27,7 @@ fat.test.container.images: kyleaure/cloudant-developer:1.0
 -buildpath: \
 	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
 	com.ibm.websphere.javaee.jsonp.1.0;version=latest,\
-	io.openliberty.org.testcontainers;version=latest
+	io.openliberty.org.testcontainers;version=latest,\
+	io.openliberty.com.google.gson;version=latest
```
Comment (KyleAure) [inline dev/com.ibm.ws.rest.handler.validator.cloudant_fat/bnd.bnd:31]: question: what is the additional dependency on gson used for? I don't see any source code updates that require gson.

**Review by KyleAure — APPROVED** [review]

File: dev/com.ibm.ws.rest.handler.validator.cloudant_fat/bnd.bnd (hunk 1 above)
Comment (tkburroughs) [reply dev/com.ibm.ws.rest.handler.validator.cloudant_fat/bnd.bnd:31]: Not a new dependency; an existing dependency that does not resolve properly for multi-threaded builds. I believe that is disabled except for local builds, and is often disabled for MacOS.  With a multi-thread build of just this component, it seems that most of the time the `io.openliberty.com.google.gson` project is not built in the correct order, so the build fails attempting to find that project.  By adding the project to the build path, this coerces the build to ensure the `io.openliberty.com.google.gson` is always processed prior to the build.gradle step that needs it.

In build.gradle, the following cloudant depenencies are defined:
```
cloudantCommonDeps  project(':com.ibm.ws.org.apache.commons.io'),
      					  project(':io.openliberty.com.google.gson')
```
And this appears to be fine for some builds, I suspect whenever multi-threaded building is disabled; however, when building just the `com.ibm.ws.rest.handler.validator.cloudant_fat` project by itself on a windows system with multi-threaded builds enabled, the build is unable to find the `io.openliberty.com.google.gson` project.

**Comment by tkburroughs** [comment]
#build

**Comment by tkburroughs** [comment]
#libby

**Pull request merged**